statsmodels/examples/ex_generic_mle_t.py

```diff
@@ -110,10 +110,6 @@ def nloglikeobs(self, params):
 8
 >>> tmp.shape
 (100, 100)
->>> np.dot(modp.exog, beta).shape
```

Comment:
Did you mean to delete this line (non error)?

Reply:
This line is an error, `beta` isnt defined at this point

Reply:
Is this a deal breaker?  We’re really close to never having to futz with invalid escapes ever again.

Reply:
It would be better to comment it. Cleans that do not fundamentally lose possible info are easier to approve.

Reply:
Beta is correctly defined on 119 and then this line is done again non-brokenly on 122.

Reply:
Oh, was hiding, then fine to remove.

Reply:
Needs to pass CI

Reply:
green